<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xiaoqu</class>
 <widget class="QWidget" name="xiaoqu">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>校区咨询</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:/image/timg.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/xiaoqubiaoti.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>1141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="midLineWidth">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="backb">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 255, 253);
font: 15pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <property name="text">
    <string>主页</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="backb_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 255, 253);
font: 15pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <property name="text">
    <string>校区资讯</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/xiaoqufengguang.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/xiaoquditu.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/xiaoquxinxi.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>610</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/zhuanyeshezhi.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/cool-background.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>201</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;仿宋&quot;;
background-color: rgb(251, 255, 252);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 255, 253);
font: 15pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>东校区</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>南校区</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>北校区</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>深圳校区</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>珠海校区</string>
    </property>
   </item>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>591</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;仿宋&quot;;
background-color: rgb(251, 255, 252);</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>line</zorder>
  <zorder>backb</zorder>
  <zorder>backb_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>textBrowser</zorder>
  <zorder>comboBox</zorder>
  <zorder>textBrowser_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>backb</sender>
   <signal>clicked()</signal>
   <receiver>xiaoqu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
